sdk/communication/AzureCommunicationChat/Source/Signaling/CommunicationSignalingClient.swift

```diff
@@ -163,4 +163,4 @@ class CommunicationHandler: NSObject, TrouterListener {
     }
 }
 
-public typealias EventHandler = (_ response: Any, _ eventId: ChatEventId) -> Void
+public typealias EventHandler = (_ response: TrouterEvent, _ eventId: ChatEventId) -> Void
```

Comment:
Now that the EventHandler contains a strongly-typed TrouterEvent, I doubt you still need the event id.